docs/ybm_api-key_create.md

```diff
@@ -0,0 +1,40 @@
+## ybm api-key create
+
+Create an API Key
+
+### Synopsis
+
+Create an API Key
+
+```
+ybm api-key create [flags]
+```
+
+### Options
+
+```
+      --description string   [OPTIONAL] Description of the API Key to be created.
+      --expiration string    [REQUIRED] The number of time units for which the API Key will be valid. 0 denotes that the key will never expire
```

Comment:
Should this be a string or an positive integer?

Reply:
The latter
I'm casting it to an int32 later
Can i take in an int directly from the input?

Reply:
Done